Dockerfile

```diff
@@ -20,23 +20,11 @@ COPY . .
 ENV NEXT_TELEMETRY_DISABLED 1
 
 # Set the build argument 
-ARG arg_env
+ARG PROFILE
 ARG NEXT_PUBLIC_BUILD_ID
 
-# Rename environment files based on arg_env TODO: This is silly to do just for the sake of one file name (.env.prod) if we could replace this monster with `mv ".env.$arg_env". .env.local`. Furthermore, PROFILE var should work just as well ;)
-RUN if [ "$arg_env" = "testnet" ]; then \
-      mv .env.testnet .env.local; \
-    elif [ "$arg_env" = "dev" ]; then \
-      mv .env.dev .env.local; \
-    elif [ "$arg_env" = "qa" ]; then \
-          mv .env.qa .env.local; \
-    elif [ "$arg_env" = "mainnet" ]; then \
-      mv .env.prod .env.local; \
-    elif [ "$arg_env" = "testnet.qa" ]; then \
-      mv .env.testnet.qa .env.local; \
-    elif [ "$arg_env" = "testnet.staging" ]; then \
-      mv .env.testnet.staging .env.local; \
-    fi
+# Rename environment files based on PROFILE 
+RUN cp .env.${PROFILE} .env.local
```

Comment:
This will fail for when PROFILE=mainnet.  the file .env.mainnet does not exist.  please reconsider changing the file name to .env.prod or the PROFILE set to "prod"

Reply:
@RootstockCollective/tokenomics @RootstockCollective/dao  I would prefer to change the file name to `mainnet`, any comments?

Reply:
@ivanIOV Any problem on using `.env.mainnet` rather than `.env.prod`?

Reply:
No issue from my end. @antomor

Reply:
yay, finally

Reply:
in fact, since you changed `arg_env` to `PROFILE`, the appropriate file is loaded automatically and so doesn't need to be copied any more.

Reply:
that's what [this bit](https://github.com/RootstockCollective/dao-frontend/blob/develop/next.config.mjs#L6) does

Reply:
But should we rely on that, what about if someone does not understand that and changes it?

Reply:
changes what

Reply:
the config? if so then they'd be breaking the app ...

Reply:
The point is that the nextjs config doesn't fail if the file is missing, so I'd prefer to let it fail in the docker file
(even if I agree with you that it's not required)

Reply:
@ivanIOV We renamed the `.env.prod` to `.env.mainnet`

Reply:
Perfect.  feel free to approve and merge.